app-server/generated_code/src/main/scala/com/azavea/rf/datamodel/latest/schema/driver.scala

```diff
@@ -0,0 +1,81 @@
+package com.azavea.rf.datamodel.latest.driver
+
+import com.azavea.rf.datamodel.latest.enums._
+import com.github.tminglei.slickpg._
+import spray.json._
+import scala.collection.immutable.Map
+
+
+/** Custom Postgres driver that adds custom column types and implicit conversions
+  *
+  * Adds support for the following column types
+  *  - JSONB
+  *  - JobStatus (enum)
+  *  - Visibility (enum)
+  *  - text[] (array text)
+  */
+trait ExtendedPostgresDriver extends ExPostgresDriver
+    with PgArraySupport
+    with PgSprayJsonSupport
+    with PgEnumSupport {
+
+  override val pgjson = "jsonb"
+
+  override val api = RFAPI
+
+  // Custom json protocol that supports Map[String, Any]
+  object RFJsonProtocol extends DefaultJsonProtocol {
+
+    implicit object AnyJsonFormat extends JsonFormat[Any] {
+      def write(x: Any) = x match {
```

Comment:
Since we don't really know all the options for metadata right now, this is mapped to `Map[String, Any]` -- not sure if there's a better way